growth/too/views.py

```diff
@@ -1235,6 +1234,12 @@ def get_json_data(plan, decam_style=True):
     else:
         ditherNorm = 1.0
 
+    start_mjd = min(time.Time(e.obstime, format='datetime').mjd for e in exposures)
+    end_mjd = max(time.Time(e.obstime, format='datetime').mjd + e.exposure_time/86400.0 for e in exposures)
```

Comment:
As long as you are using Astropy Time objects, you should use unitful exposure time expressions to avoid hard-coding the unit conversion factors.